server/controllers/convo.js

```diff
@@ -0,0 +1,83 @@
+const express = require("express");
+const router = express.Router();
+const mongoose = require("mongoose");
+mongoose.Promise = global.Promise;
+const Convo = require("../models/convo");
+
+// POST operation -- Create Convo
+module.exports.createConvo = async (req, res, next) => {
+    const { users } = req.body;
+    const alreadyExists = await Convo.findOne({
+        users: users,
+    });
+    try {
+        if (users[1]) {
```

Comment:
we should check this prior that the length is > 1 rather than using it in a try catch block